po/es.po

```diff
@@ -54,114 +53,66 @@ msgstr ""
 "elegido."
 
 #: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:14
-msgid "Available Formats:"
-msgstr "Formatos disponibles:"
+#: src/widgets/about_window.rs:82
+msgid "Features"
+msgstr "Características"
 
 #: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:16
-msgid "Hex"
-msgstr "Hex"
+#: src/widgets/about_window.rs:86
+msgid "Pick a Color"
+msgstr "Elige un color"
 
 #: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:17
-#: src/widgets/preferences.rs:311
-msgid "RGB"
-msgstr "RGB"
+#: src/widgets/about_window.rs:87
+msgid "Enter a color in Hex-Format"
+msgstr "Introduce un color en formato hexadecimal"
 
 #: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:18
-#: src/widgets/preferences.rs:319
-msgid "HSL"
-msgstr "HSL"
+#: src/widgets/about_window.rs:88
+msgid "Parse RGBA/ARGB Hex-Colors"
+msgstr "Analiza colores hexadecimales RGBA/ARGB"
 
 #: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:19
-#: src/widgets/preferences.rs:328
-msgid "HSV"
-msgstr "HSV"
+#: src/widgets/about_window.rs:89
+msgid "View colors in Hex, RGB, HSV, HSL, CMYK, XYZ and CIE-Lab format"
+msgstr "Puedes ver colores en Hex, RGB, HSV, HSL, CMYK, XYZ y CIE-Lab Format"
 
 #: data/com.github.finefindus.eyedropper.metainfo.xml.in.in:20
-#: src/widgets/preferences.rs:337
-msgid "CMYK"
-msgstr "CMYK"
+#: src/widgets/about_window.rs:90
+msgid "Customize which formats appear as well as their order"
+msgstr ""
```

Comment:
Why is there an empty line?

Reply:
I wanted to delete one thing and inadvertently forgot to put the chain back in place. I will fix it now.